main.py

```diff
@@ -4,192 +4,61 @@
 
 import os
 
-import discord
-from discord.enums import Status
-from discord.ext import commands
-from discord_slash import SlashCommand
+import interactions
 from dotenv import load_dotenv
 
+from config import DEBUG, TOKEN
 from utils import logutil
-from utils.common import DEBUG, DEBUG_DISCORD
 
 load_dotenv()
 
-# Configure logging for this (main.py) handler
-logger = logutil.initLogger("main.py")
+# Configure logging for this main.py handler
+logger = logutil.init_logger("main.py")
 
-# Configure logging for discord.py
-if DEBUG:
-    logging = logutil.getLogger("discord")
+logger.debug(
+    "Debug mode is %s; This is not a warning, just an indicator. You may safely ignore",
+    DEBUG,
+)
 
-logger.warning(f"Debug mode is {DEBUG} || Discord debug mode is {DEBUG_DISCORD}")
 
-DEV_GUILD = int(os.environ.get("DEV_GUILD"))
-TOKEN = os.environ.get("TOKEN")
-
-
-__GUILD_ID__ = [DEV_GUILD]
-
-custom_prefixes = {}
-default_prefixes = ["rf."]
-
-
-intents = discord.Intents.default()
-intents.members = True
-
-
-async def determine_prefix(bot, message):
-    "Determine prefix for the bot"
-
-    guild = message.guild
-
-    if guild:
-        logger.debug(
-            f"Custom prefix is '{custom_prefixes.get(guild.id, default_prefixes)}'")
-        return custom_prefixes.get(guild.id, default_prefixes)
-    else:
-        logger.debug(f"Using default prefix '{default_prefixes}'")
-        return default_prefixes
-
-logger.debug("Initial presence set")
-activity = discord.Game(name="rf.help • /help")
-client = commands.Bot(command_prefix=determine_prefix,
-                      case_insensitive=True,
-                      activity=activity,
-                      intents=intents,
-                      help_command=None,
-                      status=Status.idle
-                      )
-slash = SlashCommand(client, sync_commands=True, sync_on_cog_reload=True)
+client = interactions.Client(
+    token=TOKEN,
+    disable_sync=False,

```

Comment:
Also, @V3ntus, I had used your [boilerplate code](https://github.com/interactions-py/boilerplate) and I found out that `disable_sync` was set to `True` instead of `False`. Should I update this?

Also, sorry for the infinite pings lol, I know this is the third one.